src/app.html

```
@@ -0,0 +1,16 @@
+<!doctype html>
+<html
+    lang="%paraglide.lang%"
+    dir="%paraglide.textDirection%"
+    data-theme="axonotes-theme"
+>
+    <head>
+        <meta charset="utf-8" />
+        <link rel="icon" href="%sveltekit.assets%/favicon.png" />
+        <meta name="viewport" content="width=device-width, initial-scale=1" />
+        %sveltekit.head%
+    </head>
```

HTMLHint (1.5.0)

[error] <title> must be present in <head> tag.

(title-require)